wgengine/magicsock/magicsock_nonlinux.go

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
```

Comment:
We usually call these files `_default.go`, to indicate that it's the filler implementation when nothing better exists.

Also means we don't have to rename it when we add this for e.g. the BSDs, which are likely to just work with linux's code since they also support classic BPF.